### PR TITLE
shell script topic added

### PR DESCRIPTION
**Pull request by rushirg**
I Have added the Shell-Script topic and a resource.md file.

Please let me know if the resource.md file is correct as per your format or if it needs any changes.

Thank you!

### CODE REVIEW

**Comment by vicky002** [comment]
Hey @rushirg, can you please add little more resources? 

I will merge quickly then. :+1:

**Comment by rushirg** [comment]
Added cheat sheet, interview questions resources.
Thank you :+1:

File: Shell-Script/resource.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# **Shell Script**
+## **Links**
+* [Linux Shell Scripting Tutorial] (http://www.freeos.com/guides/lsst/)
```
Comment (Yathartha22) [inline Shell-Script/resource.md]: Please remove space between brackets.
Also, change heading `Links ` to `Resource`, I guess it would be more appropriate.

**Review by Yathartha22 — CHANGES_REQUESTED** [review]

File: Shell-Script/resource.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# **Shell Script**
+<<<<<<< HEAD
```
Comment (Yathartha22) [inline Shell-Script/resource.md]: Remove this.

File: Shell-Script/resource.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+## **Resources**
+*[Linux Shell Scripting Tutorial](http://www.freeos.com/guides/lsst/)
+*[Learn shell programming online](http://www.learnshell.org/)
+*[Bash cheat sheet](https://learncodethehardway.org/unix/bash_cheat_sheet.pdf)
+*[Shell scripting interview questions](https://career.guru99.com/shell-scripting-interview-questions)
+*[Solve Bash Questions](https://www.hackerrank.com/domains/shell)
+=======
```
Comment (Yathartha22) [inline Shell-Script/resource.md]: Remove it.

File: Shell-Script/resource.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+## **Links**
+* [Linux Shell Scripting Tutorial] (http://www.freeos.com/guides/lsst/)
+* [Learn shell programming online](http://www.learnshell.org/)
+* [Bash cheat sheet](https://learncodethehardway.org/unix/bash_cheat_sheet.pdf)
+* [Shell scripting interview questions](https://career.guru99.com/shell-scripting-interview-questions)
+* [Solve Bash Questions](https://www.hackerrank.com/domains/shell)
+>>>>>>> 0500936da61a68a29f00d20c3611c92c6d8a6c00
```
Comment (Yathartha22) [inline Shell-Script/resource.md]: Remove it.

**Review by Yathartha22 — CHANGES_REQUESTED** [review]

**Comment by rushirg** [comment]
@Yathartha22 Thank you, for your help. Please review the updated one too.

**Review by Yathartha22 — APPROVED** [review]

**Comment by Yathartha22** [comment]
Great work  :tada: @rushirg

**Pull request merged**